samples/highcharts/coloraxis/width-and-height/demo.details

```diff
@@ -0,0 +1,6 @@
+---
+ name: Color axis with specified width and height
```

Comment:
A bit long for the name. It will show on the tab when viewing the sample standalone. "Color axis width and height" would be sufficient.